doc/ref/cli/salt.rst

```diff
@@ -69,6 +69,20 @@ Options
     an explicit number of minions to execute at once, or a percentage of
     minions to execute on.
 
+.. option:: --batch-wait=BATCH_WAIT
+
+   Wait the specified time in seconds after each job is done before
+   freeing the slot in the batch of the next one.
```

Comment:
@dhiltonp Following up on this one, what leads you to believe that the code in question doesn't work as documented?  Thanks!

Reply:
I have dug more in to the code and it looks like batch_wait works.

batch_wait is the sort of feature that could easily be broken without us realizing it and there are no tests for it, so I looked at the source code. The function is long and complex, and the mechanism behind bwait wasn't immediately obvious. 

Instead of tracking down what all the relevant variables were, I asked if it was known to work in practice.